### PR TITLE
DDF-3297 added doPrivileged blocks for protected operations within the conditions

### PR DESCRIPTION
**Pull request by stustison**
#### What does this PR do?
Adds doPrivileged blocks that will be needed for the next version of Equinox
#### Who is reviewing it? 
@tbatie @rzwiefel @SmithJosh 
(please choose AT LEAST two reviewers that need to approve the PR before it can get merged)
#### Select relevant component teams: 
https://github.com/orgs/codice/teams
#### Choose 2 committers to review/merge the PR. 
(please choose ONLY two committers from below, delete the rest)
@coyotesqrl
@stustison
#### What are the relevant tickets?
[](https://codice.atlassian.net/browse/)
#### Checklist:
- [ ] Documentation Updated
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.

### CODE REVIEW

**Comment by stustison** [comment]
build now

File: platform/osgi/platform-osgi-conditions/src/main/java/org/codice/ddf/condition/SignerCondition.java [hunk 1]
```diff
@@ -84,7 +86,9 @@ public boolean isPostponed() {
   @Override
   public boolean isSatisfied() {
     Map<X509Certificate, List<X509Certificate>> signerCertificates =
-        bundle.getSignerCertificates(Bundle.SIGNERS_TRUSTED);
+        AccessController.doPrivileged(
+            (PrivilegedAction<Map<X509Certificate, List<X509Certificate>>>)
+                () -> bundle.getSignerCertificates(Bundle.SIGNERS_TRUSTED));
```
Comment (SmithJosh) [inline platform/osgi/platform-osgi-conditions/src/main/java/org/codice/ddf/condition/SignerCondition.java]: ❓ Does this actually require doPrivileged? Doesn't look like getSignerCertificates throws a SecurityException

Comment (stustison) [reply platform/osgi/platform-osgi-conditions/src/main/java/org/codice/ddf/condition/SignerCondition.java]: Getting the signed content from the bundle can throw one.

**Comment by stustison** [comment]
build now

**Pull request merged**